experiments/webext/tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+	"compilerOptions": {
+		"jsx": "react",
+		"target": "es2019",
+		"module": "commonjs",
+		"esModuleInterop": true,
+		"noImplicitAny": true,
+		"strict": true,
+		"strictNullChecks": true,
+		"forceConsistentCasingInFileNames": true,
+		"sourceMap": true,
+		"allowJs": true,
```

Comment:
Future note for us to clean up the various tsconfig settings. E.g. not sure if it should have allowJs. cc @jespino, @sbishel.

Reply:
I just copied this one from another file in the repo, I believe [webapp/tsconfig.json](https://github.com/mattermost/focalboard/blob/main/webapp/tsconfig.json), so none of the settings were intentionally chosen for the web extension.